api/src/services/tasks/tasks.js

```diff
@@ -3,8 +3,7 @@ import { requireAuth } from 'src/lib/auth'
 
 // Used when the environment variable REDWOOD_SECURE_SERVICES=1
 export const beforeResolver = (rules) => {
-  rules.skip({ only: ['tasks'] })
-  //rules.add(requireAuth)
+  rules.add(requireAuth)
 }
```

Comment:
Removed this rule as it was not allowing me to test w/graphql
If we want to bypass security for testing we can do that in our local .env files